skiko/src/iosMain/kotlin/org/jetbrains/skiko/SkikoUIView.kt

```diff
@@ -89,9 +90,11 @@ class SkikoUIView : UIView, UIKeyInputProtocol {
                     uiPress.key?.let {
                         pressedKeycodes.add(it.keyCode)
                     }
-                    skiaLayer?.skikoView?.onKeyboardEvent(
-                        toSkikoKeyboardEvent(press, SkikoKeyboardEventKind.DOWN)
-                    )
+                    if (uiPress.key?.keyCode != SkikoKey.KEY_BACKSPACE.value) {
```

Comment:
Is it possible to know the source of the key - virtual or physical keyboard? In case of physical keyboard, probably we should send all keys, so users can handle them.

Also, there can be not only text. For example, we can want to go to the previous screen when we press Backspace.

Reply:
Only physical keyboard events are received in this function. 
I made such a check to avoid double deletion.

Probably I should remove this check. And check for deletion elsewhere inside compose.

Reply:
this function is only called  when the physical keyboard pressed

Reply:
removed if condition

Reply:
For now it will work with double deletion of 2 last characters, when physical keyboard backspace pressed.
But, I think this bug is not a big problem and may be fixed later.

Reply:
> I think this bug is not a big problem and may be fixed later

If it is possible to fix this error at low-cost, better to make a fix now (in this PR on in the next one). Because without the fix we have physical keyboard unusable.

Will the previous approach, when we detect the backspace key, store pressed keys, and check it in `deleteBackward` will cause any issues?